src/filesystem.ts

```diff
@@ -138,3 +206,20 @@ async function addSampleData(fs: FileSystem): Promise<void> {
   await fs.mkdir({ directory: [ Branch.Private, "Video" ] })
   await fs.publish()
 }
+
+
+
+// 🔑
+
+
+/**
+ * Create or get a read key for accessing the WNFS private root.
+ */
+export async function readKey(): Promise<string> {
+  const maybeReadKey = await storage.getItem("readKey") as unknown as string
```

Comment:
```suggestion
  const maybeReadKey: string | null = await storage.getItem("readKey")
```

Reply:
Yeah, that is better. Accounts for the possible `null` 👍